### DISCUSSION TITLE
How to change SideModal position

### DISCUSSION THREAD

**Discussion by eliheron** [twilio-labs/paste#2870] [General] [answered]
Hello Team 👋🏾  I am using a side modal in our app and I am running into some issues with the css. Unfortunately the side modal doesn't extend down the page the way it does in the example. I tried changing the `position: absolute` to `position: fixed` and that fixed it. However, I do know that you cannot add classes to paste components. Is there a workaround that the team recommends in instances like this?

Video of the behavior:

https://user-images.githubusercontent.com/88729271/206281850-d4f40697-eedf-46ef-bcdd-0d7c6eb7bb3a.mov

**Comment by gloriliale** [comment, marked as answer]
Hi @eliheron! 

Hmmm that is unexpected as Side Modal should not be affected by scrolling behavior this way (take a peek at [its docs](https://paste.twilio.design/components/side-modal) to see). So that we can better figure out why this is happening, could you please provide us with a CodeSandbox reproduction? Many thanks!!

**Reply by shleewhite** [reply]
Hi @eliheron, do you still need help with this issue? Could you provide a code sandbox reproduction?

**Reply by eliheron** [reply]
Hi @shleewhite 👋🏾 No I don't still need help. Could you close the issue? I dont see a way to do it on my end. Thank you for circling back this completely slipped my mind!

**Reply by kelseycavitt** [reply]
Done, thanks @eliheron !
